helix-term/src/ui/mod.rs

```diff
@@ -640,4 +641,48 @@ pub mod completers {
             .map(|(name, _)| ((0..), name.into()))
             .collect()
     }
+
+    pub fn program(_editor: &Editor, input: &str) -> Vec<Completion> {
+        static PROGRAMS_IN_PATH: Lazy<Vec<String>> = Lazy::new(|| {
+            // Go through the entire PATH and read all files into a vec.
+            let mut vec = std::env::var("PATH")
+                .unwrap_or("".to_owned())
+                .split(":")
```

Comment:
This will work on unix but not windows. You can use https://doc.rust-lang.org/std/env/fn.split_paths.html to split the path in a crossplatform way